backend/routes/leaderboard.py

```
@@ -130,11 +130,13 @@ def get_weekly_comparison_stats():
         most_recent_monday = get_most_recent_monday()
 
         # Calculate the percentages
-        percent_shorter_streak = calculate_percent_shorter_streak(user.streak)
-        percent_fewer_modules = calculate_percent_fewer_modules(
-            user_id, most_recent_monday
+        percent_shorter_streak = round(calculate_percent_shorter_streak(user.streak))
+        percent_fewer_modules = round(
+            calculate_percent_fewer_modules(user_id, most_recent_monday)
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)